src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java

```diff
@@ -155,7 +156,8 @@ public List<AdvertisementTitleDto> getFirst10BySearchParameters(@Valid @RequestB
             consumes = {
                     MediaType.APPLICATION_JSON_VALUE,
                     MediaType.MULTIPART_FORM_DATA_VALUE,
-                    MediaType.APPLICATION_OCTET_STREAM_VALUE})
+                    MediaType.APPLICATION_OCTET_STREAM_VALUE},
```

Comment:
Did you have a chance to create a new advertisement from Swagger using that type?